benchmark/compiler/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+add_executable(compiler_benchmark basic.cc)
+
+target_link_libraries(compiler_benchmark benchmark::benchmark)
+target_link_libraries(compiler_benchmark benchmark::benchmark_main)
+target_link_libraries(compiler_benchmark sourcemeta::jsontoolkit::json)
+target_link_libraries(compiler_benchmark sourcemeta::jsontoolkit::jsonschema)
+
+add_custom_target(run_compiler_benchmark
```

Comment:
I would expect one target that runs ALL of the benchmarks in the project?

Reply:
Yeah we want to have a target to run all of the benchmarks... but I did not create this one initially for all :)

Reply:
I'll create one for all :)